pennylane_lightning/core/lightning_newAPI_base.py

```diff
@@ -98,6 +98,17 @@ def _setup_execution_config(self, config):
         Update the execution config with choices for how the device should be used and the device options.
         """
 
+    @abstractmethod
+    def dynamic_wires_from_circuit(self, circuit):
+        """From a given circuit, determine the number of wires and allocate a statevector if applicable. Circuit wires will be mapped to Pennylane default.qubit standard wire order.
```

Comment:
As we use this method for LT,
```suggestion
        """From a given circuit, determine the number of wires and allocate the underlying quantum state if applicable. Circuit wires will be mapped to Pennylane ``default.qubit`` standard wire order.
```

Reply:
Good idea!